pkg/storage/snapshot/restore_test.go

```diff
@@ -2028,6 +2075,23 @@ var _ = Describe("Restore controller", func() {
 	})
 })
 
+func expectVMUpdate(client *kubevirtfake.Clientset, vm *kubevirtv1.VirtualMachine) *int {
```

Comment:
Should we eventually do something like this for snapshot unit tests? (client assertions instead of reactors)
https://github.com/kubevirt/kubevirt/pull/11751